google-datacatalog-connectors-commons/tests/google/datacatalog_connectors/commons/datacatalog_facade_test.py

```diff
@@ -257,6 +257,47 @@ def test_upsert_tags_changed_should_succeed(self):
         datacatalog_client.create_tag.assert_not_called()
         self.assertEqual(1, datacatalog_client.update_tag.call_count)
 
+    def test_upsert_columntags_changed_should_succeed(self):
```

Comment:
```suggestion
    def test_upsert_tags_changed_column_uppercase_should_succeed(self):
```
I've proposed this change because the method under test is still `upsert_tags()`. The case sensitiveness of the column field is the test case itself.